### PR TITLE
1151: Increase logout time

### PR DESCRIPTION
**Pull request by f1sh1918**
The user will now auto logged out after 60mins (3600s)
Fixes: #1151

### CODE REVIEW

File: administration/src/AuthProvider.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const removeToken = () => window.localStorage.removeItem(LOCAL_STORAGE_KEY)
 
 const extractTokenPayload = (token: string): TokenPayload => {
   const payload: { exp: number; adminId: number } = JSON.parse(atob(token.split('.')[1]))
-  const expiry = new Date(payload.exp * 1000) // exp is in seconds, not milliseconds
+  const expiry = new Date(payload.exp * 3600) // exp is in seconds, not milliseconds.
```
Comment (michael-markl) [inline administration/src/AuthProvider.tsx:27]: I feel like this is not the right fix. The "*1000" simply converts to milliseconds.

Comment (f1sh1918) [reply administration/src/AuthProvider.tsx:27]: Ok that thats probably true so the payload.exp should be the right one :)

Comment (michael-markl) [reply administration/src/AuthProvider.tsx:27]: Yes, the expiry is set in the backend somewhere.

Comment (f1sh1918) [reply administration/src/AuthProvider.tsx:27]: found it and its already 1hour :)

**Comment by f1sh1918** [comment]
will be closed. nothing to do here

**Pull request closed without merging**